invokeai/app/services/shared/sqlite_migrator/migrations/migration_2.py

```diff
@@ -25,8 +23,6 @@ def __call__(self, cursor: sqlite3.Cursor):
         self._drop_old_workflow_tables(cursor)
         self._add_workflow_library(cursor)
         self._drop_model_manager_metadata(cursor)
-        self._recreate_model_config(cursor)
-        self._migrate_model_config_records(cursor)
```

Comment:
I'm trying to understand what's going on here. Was this *accidentally* included in both migration_2 and migration_3?

Reply:
Yes, this was initially misplaced due to a misunderstanding of how the migration process works. I can explain at greater length if you wish.

Reply:
Got it, I think that's ok. Just wanted to make sure I understood, because changes to existing migrations should be very rare.